### PR TITLE
feat: resendMultipleFilesMessage() support in useResendMessageCallback().

### PR DESCRIPTION
**Pull request by liamcho**
resendMultipleFilesMessage() support in useResendMessageCallback().

Fixes: [CLNP-1047](https://sendbird.atlassian.net/browse/CLNP-1047)

[CLNP-1047]: https://sendbird.atlassian.net/browse/CLNP-1047?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/modules/Channel/context/hooks/useResendMessageCallback.js [hunk 1]
```diff
@@ -86,6 +87,38 @@ function useResendMessageCallback({
...
+            });
+          })
+          // TODO: Handle on file info upload event.
+          // .onFileUploaded((
+          //   requestId,
+          //   index,
+          //   uploadableFileInfo,
+          //   err
+          // ) => {
+          // })
```
Comment (liamcho) [inline src/modules/Channel/context/hooks/useResendMessageCallback.js:106]: @HoonBaek 요 부분만 나중에 구현 해주세요~

Comment (HoonBaek) [reply src/modules/Channel/context/hooks/useResendMessageCallback.js:106]: Yeap, we don't handle `onFileUpload` now, so let's keep this as a comment for now

**Review by HoonBaek — COMMENTED** [review]
Could you apply the same change to this [file too](https://github.com/sendbird/sendbird-uikit-react/blob/main/src/modules/Thread/context/hooks/useResendMessageCallback.ts)?

**Review by HoonBaek — APPROVED** [review]

**Review by HoonBaek — APPROVED** [review]
LGTM, pls check test failure before merge

**Pull request merged**